react/src/components/LocationStateBreadCrumb.tsx

```diff
@@ -0,0 +1,49 @@
+import { useWebUINavigate } from '../hooks';
+import { Breadcrumb } from 'antd';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useLocation } from 'react-router-dom';
+
+export type SessionLauncherPageLocationState = {
+  alert?: {
+    type: 'success' | 'info' | 'warning' | 'error';
+    message: string;
+    description?: string;
+  };
+  from?: {
+    pathname: string;
+    label?: any;
+  };
+};
+
+const LocationStateBreadCrumb = () => {
+  const {
+    state: locationState,
+  }: {
+    state: SessionLauncherPageLocationState;
+  } = useLocation();
+  const webuiNavigate = useWebUINavigate();
+  const { t } = useTranslation();
+  return (
+    locationState.from && (
```

Comment:
```suggestion
    locationState?.from && (
```
From can be null. I got this error from here.
![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/2HueYSdFvL8pOB5mgrUQ/fae9702c-2089-445e-b8d8-dcaa6a6bf928.png)